### PR TITLE
5G1LVy9K5uapJE9uzG1NbMN4bTKJEyKLMtKGAtqdgbLDegNF

### PR DESCRIPTION
**Pull request by stakingluke**
# ASCAN Token Submission

## Introduction
Before submitting this PR, ensure you have followed all the requirements below and filled out all necessary details about your token.

## Checklist
Please review the checklist below and check each item before submitting your token information:

- [x] I have reviewed the [submission guidelines](https://www.psplist.xyz/why).
- [x] The submitted files are in the correct format and fulfill all requirements outlined below.

### General Information
- **Token Name**: 
- **Symbol**: 
- **Contract Address**: 
- **Decimals**: 

### Logo
- [ ] Logo file added to the PR
- **File Name**: `logo.png`
- **Size**: 256x256 pixels
- **File Extension**: png

### Metadata File
- [ ] Metadata file added to the PR
- **File Name**: `metadata.json`
- **File Extension**: json

### Token Information File
- [ ] Token information file added to the PR
- **File Name**: `info.json`
- **File Extension**: json

#### Required Content in `info.json`:
```json
{
  "name": "",
  "symbol": "",
  "type": "PSP22",
  "contractAddress": "",
  "decimals": 12,
  "description": "",
  "website": "",
  "sourceCode": "https://github.com/{Github_Username}/{Repository_Name}/path/to/contract",
  "links": [
    {
      "name": "X/Twitter",
      "url": ""
    },
    {
      "name": "Telegram",
      "url": ""
    },
    {
      "name": "Discord",
      "url": ""
    }
  ],
  "customTags": [
    "burnerWallet",
    "otherTag"
  ],
  "warnings": [],
  "owner": "ownerAddress",
  "minter": "minterAddress"
}
```

#### Note on Custom Tags
The `customTags` field in the info.json file is optional. This field can be used to include specific tags that describe unique characteristics or features of the token, such as burner wallet. Include custom tags only if they are relevant to the token’s functionality or usage.

#### Note on Owner and Minter
The owner and minter fields should include the addresses of the accounts that hold these capabilities within the token's smart contract. The owner is typically the account that has administrative rights over the entire contract, while the minter might be an account with the capability to issue new tokens.

### Note on Warnings
Please do not fill out the warnings field in the info.json file. This field is reserved for use by our auditing team, who will add relevant warnings such as "Mintable" or "Burnable" after thoroughly reviewing the token's smart contract. This ensures that all warnings are accurate and based on a comprehensive technical assessment.

### CODE REVIEW

**Comment by Kodziak** [comment]
Hey, the PR appears to be empty. Haven't you forgotten to attach the files?

**Comment by stakingluke** [comment]
Yes

Przemysław Paczoski ***@***.***> schrieb am Di. 13. Aug. 2024
um 13:10:

**Comment by stakingluke** [comment]
where can i find the metadata from my token?  i create it on inkwhale

**Comment by stakingluke** [comment]
Pleas check if it is ok

**Comment by Kodziak** [comment]
You're missing the link to the source code. I believe if it's InkWhale, it should be this one:
```
https://github.com/InkWhale-net/contracts/blob/feature/upgrade-op-4/contracts/psp22_standard/lib.rs
```

What's more:
1. Your wallet owns the smart contract, as the address should be filled in the “owner” field: 5Gp7WEKbbmGEDqodxTWLUfuY48au1MP81B29Rjtrp9s1yThn

**Comment by Kodziak** [comment]
Sorry, my bad. Please fill also the `minter` field with the same value as the `owner`. Then we're good to go.

**Review by Kodziak — APPROVED** [review]

File: 5G1LVy9K5uapJE9uzG1NbMN4bTKJEyKLMtKGAtqdgbLDegNF [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Staking Capital",
+  "symbol": "SC",
+  "type": "PSP22",
+  "contractAddress": "5G1LVy9K5uapJE9uzG1NbMN4bTKJEyKLMtKGAtqdgbLDegNF",
+  "decimals": 12,
+  "description": "",
+  "website": "",
+  "sourceCode": "https://github.com/InkWhale-net/contracts/blob/feature/upgrade-op-4/contracts/psp22_standard/lib.rs",
```
Comment (ggawryal) [inline 5G1LVy9K5uapJE9uzG1NbMN4bTKJEyKLMtKGAtqdgbLDegNF:9]:
```suggestion
  "sourceCode": "https://github.com/InkWhale-net/contracts/blob/feature/upgrade-op-4/op4_contracts/contracts/psp22_standard/lib.rs",
```

**Review by ggawryal — APPROVED** [review]
Source code address is slightly wrong, but otherwise LGTM

File: 5G1LVy9K5uapJE9uzG1NbMN4bTKJEyKLMtKGAtqdgbLDegNF (hunk 1 above)
Comment (stakingluke) [reply 5G1LVy9K5uapJE9uzG1NbMN4bTKJEyKLMtKGAtqdgbLDegNF:9]: hey what do i have to do?

**Comment by stakingluke** [comment]
okay, done. please check :) when will the token be verified?  Thanks.

**Pull request merged**